web-components/navbar/navbar-btn.css

```diff
@@ -0,0 +1,36 @@
+button {
```

Comment:
Whole file supposed to be deleted, or commented out, as we don't want styles from here to overwrite any others in the project